src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,18 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static final int ARRAY_SIZE = 6;
```

Comment:
The constant `ARRAY_SIZE` is correctly defined. This is good as it avoids magic numbers in the code.